yamale/command_line.py

```diff
@@ -18,22 +17,15 @@
 
 schemas = {}
 
+status = 0
 
 def _validate(schema_path, data_path, parser, strict):
     schema = schemas.get(schema_path)
-    try:
-        if not schema:
-            schema = yamale.make_schema(schema_path, parser)
-            schemas[schema_path] = schema
-        data = yamale.make_data(data_path, parser)
-        yamale.validate(schema, data, strict)
-    except Exception as e:
-        error = '\nError!\n'
-        error += 'Schema: %s\n' % schema_path
-        error += 'Data file: %s\n' % data_path
-        error += traceback.format_exc()
-        print(error)
-        raise ValueError('Validation failed!')
+    if not schema:
+        schema = yamale.make_schema(schema_path, parser)
+        schemas[schema_path] = schema
+    data = yamale.make_data(data_path, parser)
+    return yamale.validate(schema, data, strict)
```

Comment:
This is a good change, except for the return....can just let that call raise an exception.

Reply:
My answer bellow